skopt/dummy_opt.py

```diff
@@ -27,9 +27,11 @@ def dummy_minimize(func, dimensions, maxiter=1000, random_state=None):
         - an instance of a `Dimension` object (`Real`, `Integer` or
           `Categorical`).
 
-    * `maxiter` [int, default=1000]:
-        Number of iterations to find the minimum. In other words, the
-        number of function evaluations.
+    * `n_calls` [int, default=100]:
+        Number of calls to `func`.
+        If `n_random_starts` > 0, `n_calls - n_random_starts`
+        additional evaluations of `func` are made that are guided
+        by the `base_estimator`.
```

Comment:
I guess you copied and pasted too quickly ;) this does not apply for `dummy_minimize`.